test/ltest-cancelled-tests.lfe

```diff
@@ -0,0 +1,30 @@
+(defmodule ltest-cancel-tests
+  "This module is meant to test
+   the ability to recognize cancelled
+   tests (e.g.  fixture tests that
+   have bad contents).
+
+   In order to enable this test, take the
+   number one (1) out of the behaviour name
+   below, and you should see
+   \"One or more tests were cancelled.\" in
+   the runner output.
+
+   It is disabled because normally you want
+   to see the test suite finishing without
+   errors.
+  "
```

Comment:
I thought you said to make it a `docstring` doesn't a module docstring go right after defmodule?